atable/src/components/ACell.vue

```diff
@@ -20,8 +20,8 @@
 			v-if="column.cellComponent"
 			:is="column.cellComponent"
 			:value="displayValue"
-			v-bind="column.cellComponentProps">
-		</component>
+			v-bind="column.cellComponentProps" />
+		<div v-else-if="isHtmlValue" v-html="displayValue" />
```

Comment:
Is there a way to do this without adding in this extra `div`?

Reply:
Doesn't seem like there's an easy alternative, and someone had [asked for one](https://github.com/vuejs/vue/issues/7431) too. Generally you can use the `<template>` tag to avoid adding arbitrary elements, but that tag doesn't work with `v-html`. I could change it to `<span>` to avoid adding block elements though.